plugins/nevermind.py

```diff
@@ -0,0 +1,14 @@
+from errbot import BotPlugin, re_botcmd
+
+class Nevermind(BotPlugin):
+    """
+    Doesn't mind
+    """
+
+    @staticmethod
+    def nm_msg():
+        return "I'm sorry :("    
```

Comment:
I am still finding best way to test things.. 

There are two options so far:
1. Use the pytest plugin that errbot provides
2. Form staticmethods and test the output of those static methods. Instantiating the Plugin is not possible since for initialization we need the bot instance itself, which is what is done in the pytest plugin